nservicebus/sagas/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Sagas
-summary: NServiceBus uses event-driven architecture to include fault-tolerance and scalability in long-term business processes.
+summary: Maintain statefulness in distributed systems with the saga pattern and event-driven architectural principles with built-in fault-tolerance and scalability.
```

Comment:
Maybe we should mention NServiceBus here too

Reply:
It's already at 155 chars, so have to sacrifice something else. Do you have a suggestion?